charts/katalyst/charts/agent/templates/daemonset.yaml

```diff
@@ -78,6 +78,10 @@ spec:
               readOnly: true
             - mountPath: /tmp
               name: host-tmp-dir
+            {{- if and (index .Values.customArgs "orm-work-mode") (eq (index .Values.customArgs "orm-work-mode") "nri") }}
```

Comment:
Do we need to add the `orm-work-mode` parameter to `values.yaml` as well?

Reply:
The `orm-work-mode` parameter is optional; if not specified, the code defaults to 'bypass'. If necessary, we can introduce additional charts, such as `colocation-orm-nri`, to configure this parameter to 'nri'.